presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -177,14 +177,20 @@ alterRoutineCharacteristic
 
 routineBody
     : returnStatement
+    | externalBodyReference
     ;
 
 returnStatement
     : RETURN expression
     ;
 
+externalBodyReference
+    : EXTERNAL (NAME externalRoutineName)?
```

Comment:
The engine is not so concerned at how to run this since these are external. From the engine's perspective, it just needs to know which `FunctionNamespaceManager` it needs to ask to execute the function.